### DISCUSSION TITLE
Is AdMob removed?

### DISCUSSION THREAD

**Discussion by evangeloszotos** [invertase/react-native-firebase#8038] [Q&A] [answered]
Hi,

Was the AdMob support from rn-firebase removed?
I cannot find it anymore in the newer docs, but only in the older ones.

best,

**Comment by mikehardy** [comment, marked as answer]
Yes, quite a long time ago! It was removed upstream and is thus removed here.
However, it didn't die - invertase split it off and quite a lot of work has been done since, just in a different repo:

https://github.com/invertase/react-native-google-mobile-ads
https://www.npmjs.com/package/react-native-google-mobile-ads

Enjoy 🫡
